uiowa/uiowa
# Feature: Contact Info (content) block

Resolves: #1293 
Also resolves: #1425 

# To test
* Add the following sites to your multisites array in your `local.blt.yml` file and run `blt dsa`.
* `uiowa.edu` # This site should be unaffected by changes - the contact block should match stage and there should only be one of them listed on custom blocks page (https://uiowa.local.drupal.uiowa.edu/admin/structure/block/block-content)
* `brand.uiowa.edu` # This site should now have the default contact info block showing up in the footer (https://brand.local.drupal.uiowa.edu/).
* `coronavirus.uiowa.edu` # This site should now have the default contact info block showing up on the block page (https://coronavirus.local.drupal.uiowa.edu/admin/structure/block/block-content) but not showing up anywhere since it still uses uiowa_bootstrap.
  <img width="628" alt="Screen Shot 2020-04-23 at 11 33 54 AM" src="https://user-images.githubusercontent.com/709009/80124784-636d2380-8556-11ea-9f4d-5a218e3688b7.png">

# Description and request for feedback
As part of the homepage project, a contact info block was created and a block definition was included with UIDS base. When a website is created with SiteNow + the Collegiate split, this block definition is imported, but it refers to block content that doesn't exist.

This PR would fix that.

Is this the right approach? Relying on content being instantiated for a block to be defined seems awkward. Additionally, should this include an update for sites that don't already have this block defined to install it, so that when every site eventually moves over to using uids_base, it would exist already?

Additionally, this also includes a fix for a problem that stems from the fact that the current page LB template doesn't include all fields and this causes LB to generate a new template that duplicates some fields.

Looking for feedback!
